context/AuthHelper.tsx

```diff
@@ -49,3 +50,31 @@ export const loginAccount = async ({
     return error as Error;
   }
 };
+
+/**
+ * Logout the user and reset state
+ * @param props - setIsSignedIn, router, and resetUser()
+ * @param props.setIsSignedIn - set isSignedIn state
+ * @param props.router - router function
+ * @param props.resetUser - reset user state
+ */
+export const logoutFunction = async ({
```

Comment:
loginHelper or loginHandler is a much better name.

Reply:
fixed